serde_codegen_internals/src/attr.rs

```diff
@@ -270,6 +278,17 @@ impl Item {
                         }
                     }
 
+                    // Parse `#[serde(types(from = "type", into = "type", as = "type"))]
+                    MetaItem(List(ref name, ref meta_items)) if name == "types" => {
```

Comment:
I mainly just didn't want to pollute the `serde` namespace since all of the types are grouped together functionally. I will change that if that is not a concern.